tests/Unit/Options/Test_Options.cpp

```diff
@@ -533,3 +533,55 @@ SPECTRE_TEST_CASE("Unit.Options.OptionContext.default_stream",
                   "[Unit][Options]") {
   CHECK(get_output(OptionContext{}).empty());
 }
+
+namespace {
+// Use formatted inner types to make sure nested formatting works
+template <typename T>
+using In = std::array<T, 0>;
+struct FormatMap {
+  using type = std::map<In<int>, In<double>>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "{[int x0]: [double x0]}";
+};
+struct FormatVector {
+  using type = std::vector<In<int>>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "[[int x0], ...]";
+};
+struct FormatList {
+  using type = std::list<In<int>>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "[[int x0], ...]";
+};
+struct FormatArray {
+  using type = std::array<In<int>, 3>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "[[int x0] x3]";
+};
+struct FormatPair {
+  using type = std::pair<In<int>, In<double>>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "[[int x0], [double x0]]";
+};
+struct FormatUnorderedMap {
+  using type = std::unordered_map<In<int>, In<double>>;
+  static constexpr OptionString help = {"halp"};
+  static constexpr const char* const expected = "{[int x0]: [double x0]}";
+};
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Options.Format", "[Unit][Options]") {
+  const auto check = [](auto opt) noexcept {
+    using Opt = decltype(opt);
+    Options<tmpl::list<Opt>> opts("");
+    INFO(opts.help());
+    // Add whitespace to check that we've got the entire type
+    CHECK(opts.help().find("  "s + Opt::expected + "\n") != std::string::npos);
```

Comment:
I guess `Utilities/Literals.hpp` is included elsewhere. I'm fine with that, but @kidder  and @wthrowe  you might care :) (IWYU might care too)

Reply:
Fixed.  Thanks.